packages/frontend/src/components/BigNumberConfig/index.tsx

```diff
@@ -1,6 +1,15 @@
-import { Button, FormGroup, HTMLSelect, InputGroup } from '@blueprintjs/core';
+import {
+    Button,
+    FormGroup,
+    HTMLSelect,
+    InputGroup,
+    Radio,
+    RadioGroup,
+    Switch,
+} from '@blueprintjs/core';
```

Comment:
What do you think will be the plan for this to be then migrated to mantine? after V1/MVP?

Reply:
Is the last ticket in the milestone. Still being "designed" 
https://github.com/lightdash/lightdash/issues/5055